gaen-processes.md

```diff
@@ -26,7 +26,7 @@ To successfully verify infection status, the user must be at least 16 years of a
 
 The verification solution keeps track of how many verifications have been performed by the same user within the last 24 hours.
 If a user has exceeded 3 verifications in the relevant time period, they will be flagged as blocked from performing the actual notification process.
-The performed verifications are recorded using a hash of the pseudonym provided from ID-porten.
+The performed verifications are recorded using the pseudonym provided from ID-porten, hashed with HMACSHA256 with shared salt ("pepper").
```

Comment:
The solution with hash and shared salt could be not secure enough. 
If we leak the hash and shared salt then it could be pretty easy to crack the input pseudonym. 
I'd suggest changing it from `HMACSHA256` to one of password hashing algorithms like `PBKDF2`

Reply:
Created an issue to track this change here: https://github.com/folkehelseinstituttet/Fhi.Smittestopp.Verification/issues/9

Should we merge this as is, since it actually does reflect the current implementation, or keep it open until the change is implemented?

Reply:
Of course, it can be merged as long as there's an issue to decide on it (and then fix it).